opencti-platform/opencti-graphql/src/migrations/1707207977998-vocabulary-add-organization-types.js

```diff
@@ -0,0 +1,29 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
```

Comment:
you will need to rename this migration to set a more recent timestamp than the last one merged on master (1707487253148-rename-data-sharing-capability-description.js).

Reply:
OK, I'll check that everything is correct, and I'll modify it before merging to make sure I have the latest timestamp.

Reply:
Done